test/spotify_sdk_test.dart

```diff
@@ -1,26 +1,69 @@
+import 'dart:convert';
+
 import 'package:flutter/services.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:spotify_sdk/models/crossfade_state.dart';
+import 'package:spotify_sdk/platform_channels.dart';
 import 'package:spotify_sdk/spotify_sdk.dart';
 
 void main() {
-  const channel = MethodChannel('spotify_sdk');
+  const channel = MethodChannel(MethodChannels.spotifySdk);
 
   TestWidgetsFlutterBinding.ensureInitialized();
 
-  setUp(() {
-    channel.setMockMethodCallHandler((MethodCall methodCall) async {
-      return true;
-    });
-  });
+  setUp(() {});
 
   tearDown(() {
     channel.setMockMethodCallHandler(null);
   });
 
-  test('connectToSpotify', () async {
-    expect(
-        await SpotifySdk.connectToSpotifyRemote(
-            clientId: 'null', redirectUrl: 'null'),
-        true);
+  void setupMethodCall(String methodCall) {
+    channel.setMockMethodCallHandler((call) async {
+      switch (call.method) {
+        case MethodNames.connectToSpotify:
+          return true;
+        case MethodNames.getAuthenticationToken:
+          return MethodNames.getAuthenticationToken;
```

Comment:
Let's be more explicit 
```suggestion
          return "MOCKED_TOKEN";
```

Reply:
Also this test is entirely mocked so apart from increasing coverage I doubt there is any value to it.

Reply:
True still wip, don't know what the best way is to test platform channels